### PR TITLE
Adds doc and usage about the settings*.php file.

### PR DESCRIPTION
**Pull request by Ralt**

### CODE REVIEW

File: toolstacks/php/drupal/customizing-settings-php.rst [hunk 1]
```diff
@@ -19,9 +19,33 @@ Here is a default ``settings.php``:
     require_once($local_settings);
   }
 
+Any ``settings*.php`` file will be copied. It means you can use ``settings.dev.php`` and ``settings.staging.php``, then include the right one depending on environment variables.
+
+For example, you can load a development file for every development environment, by providing the ``drupal:load_development_overrides`` variable in Platform.sh UI, and using this code in the ``settings.php``:
```
Comment (pjcdawkins) [inline toolstacks/php/drupal/customizing-settings-php.rst]: Variables are not only in the UI - maybe just link to the variables section

Comment (pjcdawkins) [inline toolstacks/php/drupal/customizing-settings-php.rst]: If it's *every* development environment, then this variable would need to be in a parent and inherited. Maybe just say "any"

File: toolstacks/php/drupal/customizing-settings-php.rst [hunk 2]
```diff
@@ -19,9 +19,33 @@ Here is a default ``settings.php``:
...
+
+.. code-block:: php
+
+   <?php
+
+   if (!empty($conf['load_development_overrides'])) {
+     require_once dirname(__FILE__) . '/settings.development.php';
```
Comment (pjcdawkins) [inline toolstacks/php/drupal/customizing-settings-php.rst]: using `.development.` here and `.dev.` above

File: toolstacks/php/drupal/customizing-settings-php.rst [hunk 3]
```diff
@@ -19,9 +19,33 @@ Here is a default ``settings.php``:
     require_once($local_settings);
   }
 
+Any ``settings*.php`` file will be copied. It means you can use ``settings.dev.php`` and ``settings.staging.php``, then include the right one depending on environment variables.
+
+For example, you can load a development file for every development environment, by providing the ``drupal:load_development_overrides`` variable in Platform.sh UI, and using this code in the ``settings.php``:
+
+.. code-block:: php
+
+   <?php
```
Comment (pjcdawkins) [inline toolstacks/php/drupal/customizing-settings-php.rst]: Might be best to provide the entire settings.php file? Or don't include the opening `<?php`

Comment (Ralt) [inline toolstacks/php/drupal/customizing-settings-php.rst]: The `<?php` is for syntax highlighting. Not sure about what to do.

File: toolstacks/php/drupal/customizing-settings-php.rst (hunk 1 above)
Comment (Ralt) [inline toolstacks/php/drupal/customizing-settings-php.rst]: Changing the wording. I have a `seealso` section for the link to the variables section though.

File: toolstacks/php/drupal/customizing-settings-php.rst [hunk 4]
```diff
@@ -19,9 +19,33 @@ Here is a default ``settings.php``:
     require_once($local_settings);
   }
 
+Any ``settings*.php`` file will be copied. It means you can use ``settings.dev.php`` and ``settings.staging.php``, then include the right one depending on environment variables.
+
+For example, you can load a development file for any development environment, by providing the ``drupal:load_development_overrides`` environment variable, and using this code in the ``settings.php``:
```
Comment (pjcdawkins) [inline toolstacks/php/drupal/customizing-settings-php.rst]: how about ...

> adding this code to the end of the `settings.php` file:

and then remove the `<?php`

Comment (GuGuss) [inline toolstacks/php/drupal/customizing-settings-php.rst]: +1

**Pull request merged**